cardano-cli/src/Cardano/CLI/Options.hs

```diff
@@ -45,8 +45,8 @@ pref =
   Opt.prefs $
     mconcat
       [ showHelpOnEmpty
-      , helpEmbedBriefDesc PP.align
-      , helpRenderHelp customRenderHelp
+      -- , helpEmbedBriefDesc PP.align
```

Comment:
This is the magical configuration bit that makes the golden files still look good. The changes in https://github.com/smelc/optparse-applicative/commits/smelc/wip-cardano-cli-usecase/ are hardcoding this configuration change for now, but my plan is to make a PR to `optparse-applicative` to make this configurable.